### DISCUSSION TITLE
[Answered] Are .vercel.app domains indexed with Google by default?

### DISCUSSION THREAD

**Discussion by JavaNameConvention** [vercel/community#246] [Help] [answered]
Is there a way to get the default free vercel site (name.vercel.app) to add a cname dns to verify ownership on google console or get indexing on google in any way?

**Comment by CatsAreMLG** [comment, marked as answer]
`.vercel.app` Production Deployments are already automatically indexed. → 
![Google Results](https://user-images.githubusercontent.com/25875628/151285707-150ec8d7-3df5-4440-aa68-c2cf07ccc380.jpg)

You cannot add a cname to the .vercel.app domains.

**Reply by acfavian** [reply]
isn't work for me :C 
grupodeestudiostransfronterizos.vercel.app

any help or why?

That it uses "headless" wordpress by Hostinger, and doing request by graphql. BTW i don't think that generate the "not indexation"

**Reply by leerob** [reply]
Make sure you're using a Production build (e.g. `vercel --prod` from the CLI, or pushing to `main` branch).

https://vercel.com/docs/concepts/deployments/environments#production

**Reply by acfavian** [reply]
Thank you for help! Still isn't working.  
It is already in production. 
But I did do it again with Vercel CLI.

![image](https://user-images.githubusercontent.com/59548703/215897567-e8526902-1a14-442e-99ab-b5275bf7456a.png)
![image](https://user-images.githubusercontent.com/59548703/215897028-78c52034-e5fa-4d58-917a-1df77c6a3244.png)
![image](https://user-images.githubusercontent.com/59548703/215891704-a16708ed-790f-42d0-ac1d-b06e8402ad0d.png)
![image](https://user-images.githubusercontent.com/59548703/215895070-a2312198-0827-4089-bf2d-430be56e7603.png)

Maybe take more time, but it has been in production since 119 days aprox. 

Perhaps the error lies in the use of "SWR" in the wrong way. But it shouldn't, because queries to hostingerdomain/wp-admin/graphql data are "SSR" and "SWR" is for cache. 

Or isn't exit enough text data for indexation.

**Comment by JavaNameConvention** [comment]
Ok, thank you so much!

**Reply by Valenmetal** [reply]
I have the same problem, and its in production as well. Did you find the answer?

**Comment by Fonyuygita** [comment]
Thanks it really helps

**Comment by titan-double07** [comment]
it does not work for me though?
site:https://chisomwebdev.vercel.app/

**Reply by versecafe** [reply]
@titan-double07 It can take several days for google to index your site, it also helps to have a google search console account and verify your site so you can check the status of page indexing, detected sitemaps,  robots.txt, etc

**Reply by titan-double07** [reply]
thank you for your timely reply, really appreciate 🙇 ,
I've actually deployed this project since like January or so...
concerning the Google console, I will calm down to try it out again. because I have tried before it told me to download a file (google.html) and add it to my project folder but nothing happened.

**Reply by versecafe** [reply]
@titan-double07 you have to make sure the file is being deployed, you can also instead add a meta tag for verification if needed and it can take a few hours to detect the change.

**Reply by titan-double07** [reply]
thank you,
I just tried it out now and it worked (or at least the search console opened lol). 
I also just realised that I had placed the file (google.html) in the wrong folder, I had placed it in the app/ rather than in public/
